examples/async_transcribe_local_media_file.js

```diff
@@ -27,6 +27,7 @@ const token = require('./config/config.json').access_token;
                 'here'
             ]
         }],
+        diazation_type: 'premium', // Optional property which allows to define diarization type
```

Comment:
What is the correct spelling?

```suggestion
        diarization_type: 'premium', // Optional property which allows to define diarization type
```

Reply:
@amikofalvy fixed